data/wanderer/wanderers middle.txt

```diff
@@ -1747,6 +1747,9 @@ event "wanderers: kor sestor are aimless"
 		"required reputation" 1000
 	"reputation: Pirate" <?= -1
 	"unvisit planet" "Varu K'prai"
+	government "Navy (Oathkeeper)"
+		"attitude toward"
+			"Kor Sestor" -.1
```

Comment:
I don't see a need for this, or at the very least the attitude should be reset back to -.01 after the Navy is done dealing with the Sestor, but I'd rather just not change the attitude toward to begin with.

Reply:
It only makes sense that they like you more as you save them from annihilation by the hands of alien bots!
The thing is after you clear bots out of human space there are very little of them remaining, like you shut them down a few missions afterwards. 
I could make the reputation reset afterwards but they would appreciate you killing of the alien menace, even far away, at this point.